docs/CONFIGURATION.md

```diff
@@ -88,23 +89,24 @@ To create a new key:
 | QUERY_HANDLER_PLUGIN                             | (none)                           | "@edfi/meadowlark-opensearch-backend" or "@edfi/meadowlark-elasticsearch-backend"; if not set, `GET` queries will fail |
 | DOCUMENT_STORE_PLUGIN                            | @edfi/meadowlark-mongodb-backend | Future alternative: "@edfi/meadowlark-postgresql-back                                                    |
 | AUTHORIZATION_STORE_PLUGIN                   | @edfi/meadowlark-mongodb-backend | No alternative at this time.                                                                             |
-| **OAUTH_SERVER_ENDPOINT_FOR_OWN_TOKEN_REQUEST**  | (none)                           | Ex: http://localhost:3000/local/oauth/token                                                              |
-| **OAUTH_SERVER_ENDPOINT_FOR_TOKEN_VERIFICATION** | (none)                           | Ex: http://localhost:3000/local/oauth/verify                                                             |
+| **OAUTH_SERVER_ENDPOINT_FOR_OWN_TOKEN_REQUEST**  | (none)                           | Ex: <http://localhost:3000/local/oauth/token>                                                              |
+| **OAUTH_SERVER_ENDPOINT_FOR_TOKEN_VERIFICATION** | (none)                           | Ex: <http://localhost:3000/local/oauth/verify>                                                             |
 | **OWN_OAUTH_CLIENT_ID_FOR_CLIENT_AUTH**          | (none)                           | Client ID with role "verify-only"                                                                        |
 | **OWN_OAUTH_CLIENT_SECRET_FOR_CLIENT_AUTH**      | (none)                           | Client Secret                                                                                            |
 | OAUTH_HARD_CODED_CREDENTIALS_ENABLED             | false                            | Allow use of hard-coded tokens for bootstrapping startup of a new system or test automation              |
 | OAUTH_CLIENT_PROVIDED_TOKEN_CACHE_TTL            | 300000                           | Length of time in milliseconds to cache verified tokens, avoiding extra calls to the OAuth2 provider     |
 | OAUTH_CLIENT_PROVIDED_TOKEN_CACHE_MAX_ENTRIES    | 1000                             | Control the size of the token cache                                                                      |
 | IS_LOCAL                                         | true                             | When true, prints plain text logs. Else prints structured JSON logs                                      |
 | LOG_LEVEL                                        | info                             | Options: error, warn, info, debug                                                                        |
+| SAVE_LOG_TO_FILE                                 | false                            | When true, saves logs to console and to a file called meadowlark.log                             |
```

Comment:
Since this doesn't control saving to the console, consider rewriting with something like "When true, save log message to a file in addition to writing to the console"